lib/utils.js

```diff
@@ -411,6 +417,13 @@ var Utils = module.exports = {
     }
 
     return obj;
+  },
+
+  compileOrRunTemplate: function (template, data) {
```

Comment:
could just name it `template`, it's only internal so having a descriptive name is not as important.